auth/auth_test.go

```diff
@@ -62,7 +65,8 @@ func TestSignatureTxMiddlewareMultipleTxSameBlock(t *testing.T) {
 	ctx = context.WithValue(ctx, ContextKeyCheckTx, true)
 	kvStore := store.NewMemStore()
 	state := loomchain.NewStoreState(ctx, kvStore, abci.Header{Height: 27}, nil, nil)
-	state.SetFeature(features.IncrementNonceOnFailedTxFeature, true)
+	err = state.ChangeConfigSetting(IncrementNonceOnFailedTx, "1")
```

Comment:
Can do `loomchain.NewStoreState(...).WithOnChainConfig(cfg)` instead, fewer hardcoded strings to deal with.

Reply:
Fixed